main.go

```diff
@@ -32,15 +32,23 @@ func whoAmI() string {
 	return me.Username
 }
 
+var BuildVersion string
+
```

Comment:
Just one more small nitpick: can you move this to above the `whoAmI` function? Just wanted to keep all the variables/constants at the top of the file. Other than that, everything here looks good, thanks!

Reply:
https://github.com/samuong/alpaca/pull/81/commits/0221f5d01481fc400685fdb5e70c9d63284c3ed1